### PR TITLE
new find handler method user_by_cust

### PR DESCRIPTION
**Pull request by mayabrandi**
Using new find handler method user_by_cust to get customer. More relyable to query by customer id than email

### CODE REVIEW

File: cg/store/api/find.py [hunk 1]
```diff
@@ -29,6 +29,10 @@ def customer_by_id(self, id_: int) -> models.Customer:
     def user(self, email: str) -> models.User:
         """Fetch a user from the store."""
         return self.User.query.filter_by(email=email).first()
+        
+    def user_by_cust(self, customer_id: int) -> models.User:
```
Comment (patrikgrenfeldt) [inline cg/store/api/find.py]: please expand the name to user_by_customer for readability

File: cg/meta/invoice.py [hunk 2]
```diff
@@ -16,12 +16,12 @@ def __init__(self, db: Store, lims_api: lims.LimsAPI, invoice_obj: models.Invoic
...
 
     def prepare_contact_info(self, costcenter):
-        msg = f'Could not open/generate invoice. Contact information missing in database for customer {self.customer_obj.internal_id}. See log files.'
+        msg = f'Could not open/generate invoice. Contact information missing in Customer/User table for customer {self.customer_obj.internal_id}. See log files.'
         if costcenter.lower() == 'kth':
             contact_customer = self.db.customer('cust999')
         else:
             contact_customer = self.customer_obj
-        contact_user = self.db.user(contact_customer.invoice_contact)
+        contact_user = self.db.user_by_cust(contact_customer.id)
```
Comment (patrikgrenfeldt) [inline cg/meta/invoice.py]: please expand the name to user_by_customer for readabilit

File: cg/meta/invoice.py [hunk 3]
```diff
@@ -16,12 +16,12 @@ def __init__(self, db: Store, lims_api: lims.LimsAPI, invoice_obj: models.Invoic
         self.customer_obj  = invoice_obj.customer
 
     def prepare_contact_info(self, costcenter):
-        msg = f'Could not open/generate invoice. Contact information missing in database for customer {self.customer_obj.internal_id}. See log files.'
+        msg = f'Could not open/generate invoice. Contact information missing in Customer/User table for customer {self.customer_obj.internal_id}. See log files.'
```
Comment (patrikgrenfeldt) [inline cg/meta/invoice.py]: Why did you remove the information on which customer the problem arose? I guess that is generally helpful information?

Comment (mayabrandi) [reply cg/meta/invoice.py]: Didn't remove that. Just specifyed wich tables could be missing data. Customer id is still there!
